content/Accessibility in React: tools and practices.md

```diff
@@ -0,0 +1,150 @@
+
+### Fragments and the wrapper hell
+
+One of the classical mistakes that React developers usually make is to put a wrapper div in every component we create, for example:
+
+https://gist.github.com/nsantos16/8da556d82904423cab8e812a921d68fa
+
+This simple List component generates a div for each element, so when rendering the `ListComponent` it looks like this:
+
+https://gist.github.com/nsantos16/1c3159e118a2d8af754d2a57b5e732dd
+
+This makes the DOM confusing for visual interpretations programs that are used by people with visual disabilities.
+
+A solution for this is using the API that React provides, `</Fragment>`, let's see:
+
+https://gist.github.com/nsantos16/c097d8e791ad91ff45b0f835f5ab0219
+
+which renders this:
+
+https://gist.github.com/nsantos16/eae10c9f74bcc8ccec3357165fa1b584
+
+### Put a title to every page
+
+Add a title to all the pages of your application allowing users to recognize what is the content of the page and if it is relevant for them. Overall, people with visual, auditory, severe mobility and short-term memory disabilities benefit from it.
+
+Furthermore, in React the use of **hooks** makes this an easy job:
+
+https://gist.github.com/nsantos16/d42e2c264971373834c3718be90995a4
+
+https://gist.github.com/nsantos16/02cee7facd1f13004c38c4def1deac46
+
+### Accessibility auditing with react-axe
+
+Integrate tools on the development process like linters, formatter codes, pre-commit hooks, and other things that ensure code-quality and allow an agile implementation.
+
+[`react-axe`](<[https://github.com/dequelabs/react-axe](https://github.com/dequelabs/react-axe)>) is one of them, this linter/analyzer will improve the process of including accessibility in your application in a fast way.
+
+This tool in its latest version, apart from statically analyzing the code of your application, also _analyzes in run-time_, that means that if, for example, the gradient of a screen is not sufficient for people with visual disabilities, the linter will throw a warning when you are running your application.
+
+To integrate this tool, first, you need to add it as a dev dependency to your project
+
+```console
+yarn add react-axe --dev
+```
+
+then, you have to initialize it
+
+https://gist.github.com/nsantos16/ff2d9eb40be4541ba9af577c0dd70e6f
+
+And done! You have an accessibility linter integrated into your project.
+
+### Buttons!
+
+Buttons are a complex part of accessibility, the best way to improve buttons for accessibility is for your design team to involve practices in the development process.
+
+Here in [Rootstrap](https://www.rootstrap.com/) we have guidelines about accessibility, you can check them out [here](https://www.figma.com/file/BsH7BBDNKPBQnaKdunKtJg/RS-UI-Global-Template)!
+
+#### States
+
+![Button states](https://i.ibb.co/N7Cxy7K/Captura-de-Pantalla-2020-07-24-a-la-s-17-59-19.png)
+
+The states of the buttons in React are very important. Three states are essential for correct behavior on accessibility, these are:
```

Comment:
I think states should be singular here: `The state of buttons is very important in React`